components/layout/EntryLayout.tsx

```diff
@@ -108,16 +108,47 @@ const EntryDetails = (entry?: Entry) => (
 
 export const EntryLayout = (props: EntryLayoutProps) => {
   const { entry, carouselItems } = props
+  const [ showRollup, setShowRollup ] = useState(true);
+
   return (
     <div className="text-white">
       <EntryHeader {...entry} />
       <EntryDetails {...entry} />
       <Chart
-        rollupToPatternClass={false}
+        rollupToPatternClass={showRollup}
         showLabels={true}
         terms={entry?.terms}
         entryId={entry?._id}
       />
+      <form className="flex m-3 p-3 bg-gray-200">
+        <div className="text-black text-sm mr-2">
+          Chart by:
+        </div>
+        <div className="radio text-black text-sm mr-3">
+          <label>
+            <input
+              type="radio"
+              value="patternClass"
+              checked={showRollup === true}
+              onChange={() => setShowRollup(!showRollup)}
+              className="mr-1 ml-1"
+            />
+            Pattern class
+          </label>
+        </div>
+        <div className="radio text-black text-sm mr-3">
+          <label>
+            <input
+              type="radio"
+              value="term"
+              checked={showRollup === false}
+              onChange={() => setShowRollup(!showRollup)}
+              className="mr-1 ml-1"
+            />
+            Terms
+          </label>
+        </div>
+      </form>
```

Comment:
i added these quick radio button toggles for easier debugging (it's clumsy to compare the marker to the entry page) - i can remove them before merging this, or maybe it's actually a sorta cool feature to clean up and keep around (or tuck behind a simple feature flag for now?)
![Screenshot from 2023-01-22 13-13-58](https://user-images.githubusercontent.com/5132349/213919296-78f62178-ffe9-412a-8464-92150da84e82.png)